src/themes/elife/index.ts

```diff
@@ -39,4 +49,126 @@ ready((): void => {
       )
     )
   )
+
+  buildSocialSharers(getArticleTitle(), getArticleDoi())
 })
+
+const buildSocialSharers = (title: string, doi: string): void => {
+  const escapedTitle = encodeURIComponent(title)
+  const escapedDoi = encodeURI(`https://doi.org/${doi}`)
+
+  const attachPoint = first(':--Article :--title')
```

Comment:
Looking on how Stencila do their coding they seem to put their clases together ie. ```:--Article:--title```

Reply:
Different use case: I want a `:--title` descended from an `:--Article`.